demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py

```diff
@@ -292,6 +332,7 @@ def _search(
         content_type: Optional[ContentType] = None,
         ids_list: Optional[Iterable[int]] = None,
         all_level_dependencies: bool = False,
+        all_level_imports: bool = False,
```

Comment:
Let's change it to `all_level_relationships`, as it  provides data for all relatiohips and not just `imports`.

Reply:
On a second thought, we have to specify the relationship type in the query itself, so we will have to run the same query for all our relationships.
So let's keep that this way